### PR TITLE
[UXE-5723] feat: migrate load service to v4 digital certificates

### PR DESCRIPTION
**Pull request by pauloSF0**
### What is the new behavior introduced by this PR?
[UXE-5723] feat: migrate load service to v4 digital certificates
### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.

https://github.com/user-attachments/assets/75cfa682-f545-4b6b-8f0b-3afd377635b5

### Does it have a link on Figma?

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [ ] Application responsiveness was tested to different screen sizes
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

[UXE-5723]: https://aziontech.atlassian.net/browse/UXE-5723?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by aloisio-m-bastian — DISMISSED** [review]

**Review by aloisio-m-bastian — APPROVED** [review]

**Review by vinigfer — DISMISSED** [review]
Please update with branch freezing to see the tests execution

File: src/tests/helpers/convert-date.test.js [hunk 1]
```diff
@@ -107,4 +108,18 @@ describe('convertDate', () => {
     const dateString = 'invalid-date'
     expect(() => formatExhibitionDate(dateString, 'short', 'short')).toThrow()
   })
+
+  it('should return the correct start and end dates for the current month', () => {
+    const mockDate = new Date('2023-11-15T10:00:00Z') // Example date
```
Comment (lucasmendes21) [inline src/tests/helpers/convert-date.test.js:113]: remove esse comentario // Example date

File: src/tests/services/digital-certificates-services/v4/load-digital-certificates-service.test.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    const { sut } = makeSut()
+
+    const result = await sut({ id: fixtures.certificateMock.id })
+
+    expect(result).toEqual({
+      id: fixtures.certificateMock.id,
+      name: fixtures.certificateMock.name,
+      type: fixtures.certificateMock.type,
+      managed: fixtures.certificateMock.managed,
+      csr: undefined
```
Comment (lucasmendes21) [inline src/tests/services/digital-certificates-services/v4/load-digital-certificates-service.test.js:55]: porque tu não usou o `fixtures.certificateMock.csr`

**Review by lucasmendes21 — APPROVED** [review]
Apontei alguns pontos que não impacta mais poderia ser ajustado.

**Review by lucasmendes21 — APPROVED** [review]

**Review by aloisio-m-bastian — APPROVED** [review]

**Review by lucasmendes21 — APPROVED** [review]

**Review by lucasmendes21 — APPROVED** [review]

**Review by lucasmendes21 — APPROVED** [review]

**Review by robsongajunior — APPROVED** [review]
LGTM

**Pull request merged**